data/ElizabethOgbee.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Elizabeth Ogbe",
+  "displayStatPublic": true,
+  "type": "personal",
+  "avatar": "https://github.com/ElizabethOgbee.png",
```

Comment:
We no longer need `displayStatPublic` & `avatar` keys. So please remove it from your `json`. 

```suggestion
  "type": "personal",
```